docs/release-notes.txt

```diff
@@ -4,10 +4,103 @@
  Release Notes
 ###############
 
+**************
+ Version 0.15
+**************
+
+Version 0.15.0
+==============
+
+**Release Date:** April 14, 2021
+
+**New Features**
+
+-  WebUI: Provide historical utilization data on the Cluster page. This
```

Comment:
```suggestion
-  WebUI: Provide historical allocation data on the Cluster page. This
```

Reply:
@mackrorysd is the cloudpickle change transparent to the user?

Reply:
Yes it should be. Adding a relnote here: https://github.com/determined-ai/determined/pull/2227

Reply:
I included #2227 in this PR!